### ISSUE TITLE
Crash with RuboCop 0.47.0

### ISSUE THREAD

**Issue by oliverklee** [brigade/haml-lint#170]
When I update RuboCop from 0.46.0 to 0.47.0 on the project https://github.com/braingourmets/crud and run bin/rake haml_lint, I get a crash:

```
klee@gonzales:~/src/ruby/crud$ bin/rake haml_lint
Running via Spring preloader in process 4652
haml-lint app/views/
/home/klee/.rvm/gems/ruby-2.4.0/gems/haml_lint-0.19.0/lib/haml_lint/ruby_parser.rb:2:in `require': cannot load such file -- rubocop/ast_node/builder (LoadError)
        from /home/klee/.rvm/gems/ruby-2.4.0/gems/haml_lint-0.19.0/lib/haml_lint/ruby_parser.rb:2:in `<top (required)>'
        from /home/klee/.rvm/gems/ruby-2.4.0/gems/haml_lint-0.19.0/lib/haml_lint.rb:12:in `require'
        from /home/klee/.rvm/gems/ruby-2.4.0/gems/haml_lint-0.19.0/lib/haml_lint.rb:12:in `<top (required)>'
        from /home/klee/.rvm/gems/ruby-2.4.0/gems/haml_lint-0.19.0/bin/haml-lint:3:in `require'
        from /home/klee/.rvm/gems/ruby-2.4.0/gems/haml_lint-0.19.0/bin/haml-lint:3:in `<top (required)>'
        from /home/klee/.rvm/gems/ruby-2.4.0/bin/haml-lint:22:in `load'
        from /home/klee/.rvm/gems/ruby-2.4.0/bin/haml-lint:22:in `<main>'
        from /home/klee/.rvm/gems/ruby-2.4.0/bin/ruby_executable_hooks:15:in `eval'
        from /home/klee/.rvm/gems/ruby-2.4.0/bin/ruby_executable_hooks:15:in `<main>'
rake aborted!
Command failed with status (1): [haml-lint app/views/...]
/home/klee/src/ruby/crud/Rakefile:31:in `block in <top (required)>'
/home/klee/.rvm/gems/ruby-2.4.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:287:in `load'
/home/klee/.rvm/gems/ruby-2.4.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:287:in `block in load'
/home/klee/.rvm/gems/ruby-2.4.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:259:in `load_dependency'
/home/klee/.rvm/gems/ruby-2.4.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:287:in `load'
-e:1:in `<main>'
Tasks: TOP => haml_lint
(See full trace by running task with --trace)
```

The crash does not occur with rubocop 0.47.0.

**Comment by Bjoernsen** [comment]
Hi, since today, I retrieve the same error during my CI build.

```bash
$ gem install haml_lint --no-ri --no-rdoc
Successfully installed tilt-2.0.5

HEADS UP! Haml 4.0 has many improvements, but also has changes that may break
your application:

* Support for Ruby 1.8.6 dropped
* Support for Rails 2 dropped
* Sass filter now always outputs <style> tags
* Data attributes are now hyphenated, not underscored
* html2haml utility moved to the html2haml gem
* Textile and Maruku filters moved to the haml-contrib gem

For more info see:

http://rubydoc.info/github/haml/haml/file/CHANGELOG.md

Successfully installed haml-4.0.7
Building native extensions.  This could take a while...
Successfully installed rainbow-2.2.1
Successfully installed ast-2.3.0
Successfully installed parser-2.3.3.1
Successfully installed powerpack-0.1.1
Successfully installed ruby-progressbar-1.8.1
Successfully installed unicode-display_width-1.1.3
Successfully installed rubocop-0.47.0
Successfully installed sysexits-1.2.0
Successfully installed haml_lint-0.19.0
11 gems installed
$ haml-lint app/views/
/usr/local/lib/ruby/site_ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- rubocop/ast_node/builder (LoadError)
	from /usr/local/lib/ruby/site_ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/local/bundle/gems/haml_lint-0.19.0/lib/haml_lint/ruby_parser.rb:2:in `<top (required)>'
	from /usr/local/lib/ruby/site_ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/local/lib/ruby/site_ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/local/bundle/gems/haml_lint-0.19.0/lib/haml_lint.rb:12:in `<top (required)>'
	from /usr/local/lib/ruby/site_ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/local/lib/ruby/site_ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/local/bundle/gems/haml_lint-0.19.0/bin/haml-lint:3:in `<top (required)>'
	from /usr/local/bundle/bin/haml-lint:22:in `load'
	from /usr/local/bundle/bin/haml-lint:22:in `<main>'
ERROR: Build failed: exit code 1
```

**Comment by Bjoernsen** [comment]
https://github.com/brigade/haml-lint/blob/0276a78ecd3f3f66a11f0ee205818fa5fa6e6b89/lib/haml_lint/ruby_parser.rb#L2

`ast_node` has been renamed to `ast`, see https://github.com/bbatsov/rubocop/commit/48f1637eb36575a95b2004fecd3212d848d64f59

**Comment by sds** [comment]
Thanks for the report, @oliverklee, and for digging into the underlying cause, @Bjoernsen.

Fixed in 3ce86baf.

**Comment by oliverklee** [comment]
@sds Thanks! :heart:
